### PR TITLE
implemented #141

### PR DESCRIPTION
**Pull request by JanEickholt**
usage in config:
"weapons": "Vandal, Phantom, ..."
cmd_9NCHKzXrke

### CODE REVIEW

**Comment by JanEickholt** [comment]
I decided to remove the redundent "Vandal" or "Phantom" at the end of the Skin name, to fit more weapons.
If you want to remove this on a final release you can change line 50 in Loadouts.py to:
weaponLists[players[player]["Subject"]][weapon["displayName"]] = (color(skin["displayName"], fore=rgb_color))

**Comment by JanEickholt** [comment]
Otherwhise you could think about cutting it even further-
Sentinels of Light = SOL
Gravitational Uranium Neuroblaster = GUN
Radiant Entertainment System = RES

**Comment by zayKenyon** [comment]
Putting this on hold whilst the team discuss this. There are a couple changes made here which I'm not sure about. 

Firstly the matter of config, we need to make sure users' inputs for the weapon(s) is a smooth process. Secondly, I'm hesistant to employ abbreviations, it is not something I want to put pressure on us to maintain nor do I want to push us for more often releases just to update this list; I'm sure an argument could be made for the app to request a list of abbreviations from our repo instead of us re-releasing copies of the app, but that's not something I feel particularly strong for either.

This will be a good PR, we just need to consider other approaches for implementing this.

File: src/constants.py [hunk 1]
```diff
@@ -154,8 +154,24 @@
...
+                "VALORANT GO! Vol. 1": "V. G. Vol. 1",
+                "VALORANT GO! Vol. 2": "V. G. Vol. 2",
+                "Radiant Crisis 001": "RC 001",
+                "Sentinels of Light": "SoL",
+                "Prelude to Chaos": "PtC",
+                "Winterwunderland": "Winter",
+                "Kohaku & Matsuba": "K&M",
+                "Gaia's Vengeance": "Gaia"
+            }
+        },
```
Comment (zayKenyon) [inline src/constants.py:174]: See: https://github.com/zayKenyon/VALORANT-rank-yoinker/pull/161#issuecomment-1658976073

**Comment by JanEickholt** [comment]
> Firstly the matter of config, we need to make sure users' inputs for the weapon(s) is a smooth process.
I believe, that just seperating them with a comma is quite a smooth process, especially if there are 2 weapons defaulted.

> Secondly, I'm hesistant to employ abbreviations
I compleatly understand your position here, I just thought that "Gravitational Uranium Neuroblaster Operator" is quite a long name for a columne, which is why i would shorten some, or at least remove the weapon tag "Operator" at the end.

 > This will be a good PR, we just need to consider other approaches for implementing this.
I appreciate to hear that :D

**Comment by nanometer5088** [comment]
> Otherwhise you could think about cutting it even further- Sentinels of Light = SOL Gravitational Uranium Neuroblaster = GUN Radiant Entertainment System = RES

I personally disagree with this. Unofficial abbreviations would be very confusing for a lot of people.

**Comment by JanEickholt** [comment]
> I personally disagree with this. Unofficial abbreviations would be very confusing for a lot of people.

I somewhat agree with that statement, but like I said: "Gravitational Uranium Neuroblaster Operator" is quite long for a columne and normally takes up multiple rows to be displayed.
I sadly don't know any official abbreviations, which is why i just chose some "random" - Prelude to Chaos is normally just called "Chaos Vandal"... not PtC

**Pull request closed without merging**